### DISCUSSION TITLE
Frontend contribution focus

### DISCUSSION THREAD

**Discussion by stackoverfloweth** [open-sauced/feedback#11] [Feature request]
Could you have filters for frontend frameworks like `vue`? I'm also extremely interested in building/contributing to component libraries, maybe that's something else that could be highlighted by opensauced?

**Comment by bdougie** [comment]
I opened an issue for this to track. Kubernetes also came up as an option after I appeared on the [bret.live](https://www.youtube.com/watch?v=ta-btidg_SM) podcast.

https://github.com/open-sauced/insights/issues/1355

Contribution is welcomed

**Comment by Ibinola** [comment]
Would also love if there's something for `Angular` as well
